kubespawner/__init__.py

```diff
@@ -12,6 +12,6 @@
 # Users can simply import kubespawner.KubeSpawner in their applications
 # instead of the more verbose import kubespawner.spawner.KubeSpawner.
 
-from kubespawner.spawner import KubeSpawner
+from kubespawner.spawner import KubeSpawner, __version__
```

Comment:
nit unrelated to this PR: do you know why we use `from kubespawner.spawner import KubeSpawner` here? Normally inside a package I'd expect to see relative imports so `from .spawner import KubeSpawner`.

Reply:
No clue, I lack the python experience to say anything about it.